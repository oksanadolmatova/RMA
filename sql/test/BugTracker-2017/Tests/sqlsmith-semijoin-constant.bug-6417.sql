select  
  ref_0.active as c0
from 
  sys.sessions as ref_0
    right join sys.netcdf_dims as sample_5
    on ((true) 
        or (ref_0.active is NULL))
where ref_0.querytimeout is NULL
limit 106;
