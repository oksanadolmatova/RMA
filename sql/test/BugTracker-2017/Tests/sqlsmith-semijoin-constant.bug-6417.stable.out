stdout of test 'sqlsmith-semijoin-constant.bug-6417` in directory 'sql/test/BugTracker-2017` itself:


# 19:24:14 >  
# 19:24:14 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=31056" "--set" "mapi_usock=/var/tmp/mtest-31153/.s.monetdb.31056" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/niels/scratch/rc-monetdb/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2017"
# 19:24:14 >  

# MonetDB 5 server v11.27.6
# This is an unreleased version
# Serving database 'mTests_sql_test_BugTracker-2017', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 7.330 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2017 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://localhost.nes.nl:31056/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-31153/.s.monetdb.31056
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded

Ready.

# 19:24:14 >  
# 19:24:14 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-31153" "--port=31056"
# 19:24:14 >  

#select  
#  ref_0.active as c0
#from 
#  sys.sessions as ref_0
#    right join sys.netcdf_dims as sample_5
#    on ((true) 
#        or (ref_0.active is NULL))
#where ref_0.querytimeout is NULL
#limit 106;
% .L7 # table_name
% c0 # name
% boolean # type
% 5 # length

# 19:24:14 >  
# 19:24:14 >  "Done."
# 19:24:14 >  

